### ISSUE TITLE
Cannot run action on branch delete event

### ISSUE THREAD

**Issue by asizikov** [pulumi/actions#41]
I'm trying to implement the following scenario:

* On PR create a new stack, deploy provision resources with `pulumi up` and deploy my code
* On feature branch delete event run `pulumi destroy` and `pulumi stack rm` to clean things up.

I have this github action:

```yaml
name: 'Clean up resources provisioned for PR'

on: 
  delete

jobs:
  clean:
    if: github.event.ref_type == 'branch'
    runs-on: ubuntu-latest
    name: 'Clean up Pulumi resources'
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 1      
      - name : 'Prepare stack name'
        run: |
           deletedBranchName=${{ github.event.ref }}
           stackName=${deletedBranchName//[\/]/-}
           echo $stackName
           echo "STACK_NAME=$stackName" >> $GITHUB_ENV
      - uses: pulumi/actions@v1
        with:
          command: destroy 
        env:
          ARM_CLIENT_ID: ${{ secrets.ARM_CLIENT_ID }}
          ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
          ARM_TENANT_ID : ${{ secrets.ARM_TENANT_ID  }}
          PULUMI_ROOT: infra
          PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
          PULUMI_STACK_NAME: ${{ env.STACK_NAME }}
          IS_PR_WORKFLOW: false
```

However whenever i'm trying to run it doesn't do anything except logging this message:

```
/usr/bin/docker run --name cc4956b3361ac347eb495da318693f749e4ece_67c970 --label cc4956 --workdir /github/workspace --rm -e STACK_NAME -e ARM_CLIENT_ID -e ARM_CLIENT_SECRET -e ARM_TENANT_ID -e PULUMI_ROOT -e PULUMI_ACCESS_TOKEN -e PULUMI_STACK_NAME -e IS_PR_WORKFLOW -e INPUT_COMMAND -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/pulumi-azure/pulumi-azure":"/github/workspace" cc4956:b3361ac347eb495da318693f749e4ece  "destroy"
Logging in using access token from PULUMI_ACCESS_TOKEN
Logged in to pulumi.com as [my-name](https://app.pulumi.com/[my-name])
PR event (null) contains no changes and does not warrant a Pulumi Preview
Skipping Pulumi action altogether...
```
I guess the problem is in this line:
`PR event (null) contains no changes and does not warrant a Pulumi Preview`

Indeed, there are no changes in this PR... it's not even a PR, but a `delete` event.

**Comment by viveklak** [comment]
Looks like https://github.com/pulumi/actions/blob/6756529833977db27f4e08f45d801574e996e5de/entrypoint.sh#L50 isn't really meant to handle deletion. Assigning to @stack72 to see if this is something that can be supported.

**Comment by stack72** [comment]
This won't be fixed until #40 has been put in place

**Comment by stack72** [comment]
I am closing this out as we have just released v2 of our Action which is based on TypeScript and addresses this issue
